### PR TITLE
just adding a vimplugin.

### PR DESCRIPTION
**Pull request by brettlyons**

### CODE REVIEW

File: pkgs/misc/vim-plugins/generated.nix [hunk 1]
```diff
@@ -2157,6 +2157,17 @@ let
     };
   };
 
+  notational-fzf-vim = buildVimPluginFrom2Nix {
+    pname = "notational-fzf-vim";
+    version = "2019-12-03";
+    src = fetchFromGitHub {
```
Comment (jonringer) [inline pkgs/misc/vim-plugins/generated.nix]: the update script will automatically determine the rev based on the date. Please change commits message to:
```
vimPlugins.notational-fzf-vim: init at 2019-12-03
```

**Review by jonringer — COMMENTED** [review]
otherwise LGTM

File: pkgs/misc/vim-plugins/generated.nix (hunk 1 above)
Comment (brettlyons) [reply pkgs/misc/vim-plugins/generated.nix]: Fixed :)

**Comment by jonringer** [comment]
please clean up git history, should only have 2 commits

**Comment by brettlyons** [comment]
> please clean up git history, should only have 2 commits

How do I do that?

**Comment by jonringer** [comment]
the "easiest" way would be be to pop off the commits and fix the message

```
git reset HEAD~2
git add pkgs/
git commit -m "vimPlugins.notational-fzf-vim: init at 2019-12-03"
git push .. .. --force
```

**Comment by brettlyons** [comment]
I used git commit --amend "new msg" after resetting to head - 2, then force pushed. git log shows two commits, hash's ed8bfc9 and f4ac439.  If it gets cleaner than that, let me know how.

**Review by jonringer — APPROVED** [review]
diff LGTM
commits LGTM

```
[35 built, 0.0 MiB DL]
https://github.com/NixOS/nixpkgs/pull/80368
17 package built:
vimPlugins.airline vimPlugins.coc-git vimPlugins.denite vimPlugins.fugitive vimPlugins.neoterm vimPlugins.notational-fzf-vim vimPlugins.nvim-lsp vimPlugins.riv vimPlugins.vim-airline-themes vimPlugins.vim-localvimrc vimPlugins.vim-ps1 vimPlugins.vim-rooter vimPlugins.vim-slime vimPlugins.vim-test vimPlugins.vim-themis vimPlugins.vim-visual-multi vimPlugins.vimoutliner
````

**Pull request merged**